charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/csi-azuredisk-storage-class.yaml

```diff
@@ -0,0 +1,154 @@
+{{- if .Values.storageClasses.create -}}
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ .Values.storageClasses.storageClassNames.standardLRS }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
storage class is not a namespaced k8s object.

Reply:
Fixed.